homework/shelkovich_ilya/arrayService/src/test/java/com/tutrit/java/quickstart/service/ArrayServiceTest.java

```diff
@@ -0,0 +1,62 @@
+package com.tutrit.java.quickstart.service;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ArrayServiceTest {
+
+    ArrayService arrayService;
+    Logger log = LoggerFactory.getLogger(ArrayServiceTest.class);
+
+
+    @Before
+    public void setUp() {
+        this.arrayService = new ArrayService();
+    }
+
+    @Test
+    public void createArray() {
+        Assert.assertArrayEquals(new int[]{}, arrayService.createArray(0));
+    }
+
+    @Test
+    public void fillRandomly() {
+        int[] array = new int[10];
+        Assert.assertNotNull(arrayService.fillRandomly(array));
+    }
```

Comment:
the array after creation will not be null. all of its elements will be equal to 0. this is an int array. Or I'm wrong?

Reply:
I think You need check size of array.

Reply:
fixed